galasa-extensions-parent/dev.galasa.ecosystem.builder/src/main/resources/docker/config.properties

```diff
@@ -0,0 +1,67 @@
+subnet=172.21.0.0/16
```

Comment:
Transient files should not be committed.
If this is a default responses files,  then should be yaml